### PR TITLE
Update motd & Login/Register links

### PR DESCRIPTION
**Pull request by atodorov**
This is a small usability improvement but I've had several users tell me those items on the screen were confusing to them.

### CODE REVIEW

File: tcms/templates/tcms_base.html [hunk 1]
```diff
@@ -112,6 +112,7 @@
 			{% if AUTH_BACKEND.can_login %}
 			<a href="{% url "django.contrib.auth.views.login" %}?next={{ request.path }}" class="banner_login">Login</a>
 			{% endif %}
+                        {% if AUTH_BACKEND.can_login and AUTH_BACKEND.can_register %} or {% endif %}
```
Comment (tkdchen) [inline tcms/templates/tcms_base.html]: Tab is used as intention in all templates, not spaces.

**Comment by tkdchen** [comment]
What is your use cases of ``product.py``. Generally, in production environment, it is recommended to use a customized ``product.py`` to configure instead of original one installed.

**Comment by atodorov** [comment]
> What is your use cases of product.py? Generally, in production environment, it is recommended to use a customized product.py to configure instead of original one installed.

This doesn't mean the default config should not provide sane default settings. By default Kerberos authentication is not enabled and should not be mentioned in a such visible place as the login screen. 
The change is generic enough for any environment. 

I've added back the tabs.

**Comment by tkdchen** [comment]
This is updated in #285

**Comment by tkdchen** [comment]
#285 is merged.

**Pull request closed without merging**